src/nft/components/explore/Table.tsx

```diff
@@ -188,7 +183,12 @@ export function Table<D extends Record<string, unknown>>({
             >
               {row.cells.map((cell, cellIndex) => {
                 return (
-                  <td className={clsx(styles.td, classNames?.td)} {...cell.getCellProps()} key={cellIndex}>
+                  <td
+                    className={styles.td}
+                    {...cell.getCellProps()}
+                    key={cellIndex}
+                    style={{ maxWidth: cellIndex === 0 ? (isMobile ? '300px' : '360px') : '160px' }}
```

Comment:
non-blocking nit. It would be nice to have constant names for these px amounts, such as MOBILE_ROW_WIDTH DESKTOP_ROW_WIDTH, etc